platform/ui/src/components/ThumbnailList/ThumbnailList.tsx

```diff
@@ -36,6 +36,7 @@ const ThumbnailList = ({
           imageSrc,
           messages,
           imageAltText,
+          isHydrated,
```

Comment:
it is kind of weird that we have `isHydrated` even for regular imagees (e.g., CT, PT, etc.), can we rename it to something else? I don't know what but basically it is `is this display set hydrated given that it is derived (rt, seg, sr)`

Reply:
How about calling it `isADerivedAndHydratedDisplaySet`?

Reply:
How about `isHydratedForDerivedDisplaySet`